app/models/cart.rb

```diff
@@ -15,4 +15,8 @@ def find_by_course_id
   def add_course(id)
     content[id.to_s] = (content[id.to_s] || 0) + 1
   end
+
+  def subtract_course(id)
```

Comment:
Let's discuss refactor to account for when there is only 1 of an item; in such case item needs to be deleted completely from cart; make sure item doesn't show in cart when count is a negative value @yoasyo25  @thedanielvogelsang